dependencies/setupFmi4c.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Shell script building Hopsan dependency fmi4c
+
+basedir=$(pwd)
+
+name=fmi4c
+codedir=${basedir}/${name}-code
+builddir=${basedir}/${name}-build
+installdir=${basedir}/${name}
+
+
+# Download and verify
+./download-dependencies.py ${name}
+
+# Include general settings
+source setHopsanBuildPaths.sh
+
+# Create build dir and enter it
+mkdir -p $builddir
+cd $builddir
+
+# Generate makefiles
+cmake -Wno-dev -G %HOPSAN_BUILD_CMAKE_GENERATOR% -DCMAKE_INSTALL_PREFIX=${installdir} ${codedir}
+
+# Build and install
+cmake --build . --parallel 8
+cmake --build . --target install all
```

Comment:
I think target install should be enough ?
But i did not check your cmake file, so I dont know

Reply:
Right, I will remove target "all".